README.md

```diff
@@ -62,6 +62,17 @@ This demonstration will create a number of resources.
 
 ## Prerequisites
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```

Comment:
[Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)

Reply:
please test this

Reply:
Looks great